source/limitations.txt

```diff
@@ -0,0 +1,96 @@
+.. _entity-framework-limitations:
+
+===========
+Limitations
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: EF, EF Core, support
+
+Overview
+--------
+
+On this page, you can find a list of MongoDB and {+framework+} features that
+are not supported by the {+provider-long+}. Because the provider is in active
+development, some features listed on this page might be considered for future
+releases based on customer demand.
+
+Unsupported MongoDB Features
+----------------------------
+
+The following sections describe MongoDB features that are not supported by the {+provider-short+}.
+
+Time Series
+~~~~~~~~~~~
+
+The {+provider-short+} does not support time series data or time series
+collections. These are MongoDB-exclusive features that are not available in {+framework-core+}.
+
+Atlas Search
+~~~~~~~~~~~~
+
+The {+provider-short+} does not support MongoDB Atlas search. Atlas Search is a
+MongoDB-exclusive feature with no similar feature available in {+framework-core+}.
+
+Vector Search
+~~~~~~~~~~~~~
+
+The {+provider-short+} does not support MongoDB Atlas Vector search. Atlas Vector Search is a
+MongoDB-exclusive feature with no similar feature available in {+framework-core+}.
+
+Unsupported {+framework-core+} Features
+---------------------------------------
+
+The following sections describe {+framework-core+} features that are not
+supported by the {+provider-short+}.
+
+Migrations
+~~~~~~~~~~
+
+Migrations in {+framework-core+} are designed for relational databases. Because
+MongoDB is a document database, migrations are not supported by the
+{+provider-long+}.
+
+Database-First Development
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB is designed to be flexible and does not require a database schema.
+Because of MongoDB's schemaless design, database-first development is not supported
+in the {+provider-short+}. 
+
+Foreign Keys
+~~~~~~~~~~~~
+
+Because MongoDB is a document database, the {+provider-short+} does not support foreign keys.
+
+Alternate Keys
+~~~~~~~~~~~~~~
+
+Because MongoDB is a document database, the {+provider-short+} does not support alternate keys.
+
+Table Splitting
+~~~~~~~~~~~~~~~
+
+MongoDB is a document database and does not have normalized tables.
+Because of this, table splitting is not supported by the
+{+provider-short+}.
+
+Temporal Tables
+~~~~~~~~~~~~~~~
+
+The {+provider-short+} does not support temporal tables.
+
+Spatial Data
+~~~~~~~~~~~~
+
+The {+provider-short+} does not support spatial data.
```

Comment:
Q: are there any further reading links that you can include on this page? just a suggestion - ignore if you feel like it's not the pattern being used in these docs or if there arent any useful links.

Reply:
I'm not sure if links would be particularly useful on this page, since they would be linking to information that doesn't apply to the provider. I'll leave it as is for now but definitely open to adding them if a need/request comes up for more information